docs/tutorials/add-full-text-search-to-your-olab-db.md

```diff
@@ -0,0 +1,246 @@
+---
+title: Add full-text search to Clickhouse
+sidebar_position: 2
+---
+
+
+This guide will help you add full-text search to a well-known OLAP database, Clickhouse, using the Quickwit search streaming feature. Indeed Quickwit exposes a REST endpoint that streams ids or whatever attributes matching a search query **extremely fast** (up to 50 million in 1 second), and Clickhouse can easily use them with joins queries. 
+
+We will take the [Github archive dataset](https://www.gharchive.org/), which gathers more than 3 billion Github events: `WathEvent`, `PullRequestEvent`, `IssuesEvent`... You can dive into this [great analysis] (https://ghe.clickhouse.tech/) made by Clickhouse to have a good understanding of the dataset. We also took strong inspiration from this work, and we are very grateful for them to have shared this.
+
+## Install
+
+```bash
+curl -L https://install.quickwit.io | sh
+cd quickwit-0.2
```

Comment:
```suggestion
cd quickwit-v0.2.0
```

Reply:
We can also do `cd quickwit-v*/`. Whatever you decide, please let me know so we can be consistent in other parts of the docs.

Reply:
let's go for `cd quickwit-v*/`